libr/core/cbin.c

```diff
@@ -3636,7 +3636,7 @@ static int r_core_bin_file_print(RCore *core, RBinFile *binfile, int mode) {
 			RBinInfo *info = obj->info;
 			ut8 bits = info ? info->bits : 0;
 			const char *arch = info ? info->arch : "unknown";
-			r_cons_printf ("{\"objid\":%d,\"arch\":\"%s\",\"bits\":%d,\"binoffset\":%"
+			r_cons_printf ("{\"binfd\":%d,\"arch\":\"%s\",\"bits\":%d,\"binoffset\":%"
```

Comment:
i think we should use bd, as in bin descriptor. we have fd for file descriptors in rio, so it makes sense to use just 'bd' or maybe 'id' but that can be confusing because there can be other ids.